src/components/Filters/Filters.test.tsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import selectEvent from 'react-select-event';
+
+import Filters from './Filters';
+import { mockTestFields, mockTestState } from './mocks/options';
+
+const onApplyFnMock = jest.fn();
+const onCloseFnMock = jest.fn();
+
+describe('Filters', () => {
+  it('should display remove button when value exists', () => {
+    render(<Filters fields={mockTestFields} onApply={onApplyFnMock} />);
```

Comment:
There is a `renderWithProviders` in `src/utils/tests/renderWithProviders.tsx` please use it instead of `render` from `testing-library`